src/iop/basecurve.c

```diff
@@ -340,14 +340,14 @@ static void set_presets(dt_iop_module_so_t *self, const basecurve_preset_t *pres
 void init_presets(dt_iop_module_so_t *self)
 {
   // sql begin
-  DT_DEBUG_SQLITE3_EXEC(dt_database_get(darktable.db), "BEGIN", NULL, NULL, NULL);
+  //DT_DEBUG_SQLITE3_EXEC(dt_database_get(darktable.db), "BEGIN", NULL, NULL, NULL);
```

Comment:
you've removed the transaction to ensure atomic commit.

Reply:
I don't fully understand what this does, but other modules don't use that or have it commented, so I thought it was merely a debug thing. It doesn't seem to cause issues.

Reply:
The transaction ensure that every changes in the DB  between the BEGIN / COMMIT (or ROLLBACK) is either fully committed or nothing is changed.